llarp/rpc/rpc_server.cpp

```diff
@@ -1,5 +1,5 @@
 #include "rpc_server.hpp"
-#include <router/abstractrouter.hpp>
+#include <router/router.hpp>
```

Comment:
If things need to be added to a class' interface, that should be done via the interface class.  If that feels cumbersome, probably a sign that kitchen sinking is happening.

Reply:
done, squashed and rebased